tests/unit/introspection.tcl

```diff
@@ -61,6 +65,26 @@ start_server {tags {"introspection"}} {
         $rd4 close
     }
 
+tags {"needs:debug"} {
+    test {CLIENT KILL AGE will kill old clients} {
+            set rd1 [redis_deferring_client]
+            r debug sleep 2
+            set rd2 [redis_deferring_client]
+            set connected_clients [s connected_clients]
+            assert {$connected_clients >= 3}
+            # Should kill everything except r and rd2
+            set res [r client kill age 1]
```

Comment:
why won't it kill `r`?
p.s. we do have an implicit `r reconnect` feature, but i hate to rely on it, so i'd rather not disconnect `r`.
we can use `skipme`, but maybe we'll combine this with the `user` option? i.e. make both rd1 and rd2 use a specific user?

Reply:
Because `SKIPME` defaults to yes.

We could probably add a user-based filter if you're worried. I was trying to keep it simple.

Reply:
Ohh. I'm ok keeping it as is, but maybe an explicit SKIPME or a USER can make it more readable

Reply:
Actually, now that I look at it. If we kill the only two clients that are both old and use that user so we can't be sure if the age works. We need another one that uses the same user but doesn't match the age criteria.

Reply:
I think you've gotten a bit turned around. :slightly_smiling_face:  The test kills `rd1` _but not_ `rd2`.

I did change it to use a dummy user, and also just added a small fix to the test to account for that.